core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -94,6 +94,12 @@ object Invoker {
         id
       }
       .getOrElse {
+        if (config.redisHostName == "" || config.redisHostPort == "") {
```

Comment:
`== ""` could be `.isEmpty (or .nonEmpty for the opposite).
trim.isEmpty to avoid blanks?

Reply:
changed to use trim.isEmpty